ProjFS.Mac/PrjFSKext/KauthHandler.cpp

```diff
@@ -549,8 +562,13 @@ KEXT_STATIC int HandleVnodeOperation(
         {
             if (FileFlagsBitIsSet(currentVnodeFileFlags, FileFlags_IsEmpty))
             {
-                // Prevent system services from hydrating files as this tends to cause deadlocks with the kauth listeners for Antivirus software
-                if (!TryGetVirtualizationRoot(&perfTracer, context, currentVnode, pid, CallbackPolicy_UserInitiatedOnly, &root, &vnodeFsidInode, &kauthResult, kauthError))
+                bool isWriteOperation = ActionBitIsSet(action, KAUTH_VNODE_WRITE_ATTRIBUTES | KAUTH_VNODE_WRITE_EXTATTRIBUTES | KAUTH_VNODE_WRITE_DATA | KAUTH_VNODE_APPEND_DATA);
```

Comment:
nit: this line is a bit long.  Can we put the parameters for ActionBitIsSet on a separate line?

Reply:
Done!